Update design_card_template.md (#1792)
cds-snc/design-gc-conception: .github/ISSUE_TEMPLATE/design_card_template.md

---
name: Design Card template
about: Template to use when creating generic Design checklist

> title: 'Design'

Reviewer: ```suggestion
title: 'Design: '
```

> labels: ''

Reviewer: ```suggestion
labels: 'design'
```

automatically labels the task

assignees: ''

---
### Summary 
This template is to use and edit when creating a generic design checklist. For new component design , please refer to [this template](https://docs.google.com/document/d/1n-5R--fhFXdKjdSOi4VGJuiEM7jUc2JSHc6dk6t1rP4/edit?tab=t.uvjf9f6t7r01#bookmark=id.nu1pa7six16u)

### 📇 User story

_Edit this BEFORE GROOMING when creating the card_ 

As a ___,
I want to __,
so that ___.

> ### ✅Definition of Done / Outcomes
> _List what changes need to happen to meet the acceptance criteria_
> - [ ] 
> - [ ] 
> 
> _List the steps needed to meet the acceptance criteria_
> - [ ] 
> - [ ] 
> 

Reviewer: ```suggestion
### ✅ Definition of Done / actions
_Steps that need to happen to complete the task_
- [ ] 
- [ ] 

_The following outputs are created_
- [ ] 
- [ ]
```

> ### 📜Acceptance criteria
> _Add here the list of criteria that the task needs to meet before being considered done and complete_
> 
> - [ ] 
> - [ ] 
> _List the outputs_ 
> 
> - [ ] 
> - [ ] 
> - [ ] 
> _List any standards/templates to follow_
> 
> - [ ] 
> - [ ] 
> - [ ] 
> _List the reviews needed_
> 
> - [ ] 
> - [ ] 
> - [ ] 

Reviewer: ```suggestion
### 📜 Acceptance criteria / checks
_User is able to_
- [ ] _see this_
- [ ] _do that_

_The following standards/templates were followed_
- [ ] 
- [ ] 

_The following reviews happened_
- [ ] 
- [ ] 
```

### 📝 More info

> Add here any context, conversations, images, documents or links that could be useful to understand the work

Reviewer: ```suggestion
_Add here any context, conversations, images, documents or links that could be useful to understand the work_
```

> #### 🚫Out of scope
> If possible, describe items not considered part of this work

Reviewer: ```suggestion
#### 🚫 Out of scope
_If possible, describe items not considered part of this work_
```
